src/pages/components/leadspace.mdx

```diff
@@ -29,7 +28,22 @@ import ComponentFeedback from "components/ComponentFeedback";
 
 This is the most commonly used Lead space style. The left-aligned type draws the users' attention to the Lead space messaging immediately. Used in marketing and product pages, among others.
 
-![Lead space left-aligned](../../images/pattern/leadspace/Leadspace_Demo_Centered.jpg)
+There are 4 available heights:
+
+- Super (640px)
+- Tall (560px)
+- Medium (480px)
+- Short (320px)
```

Comment:
Can we provide this information in a table? Is this only for left aligned?

Reply:
for left-aligned only

Reply:
Added them in a small table, thank you, @photodow, for the idea!

Reply:
Agreed with @photodow  on **"Generally speaking, I think we should slow down on this, and only add what we have in production to this component documentation."**

Reply:
@photodow The Lead space page is one of the oldest components of websites, and we do need to work on the hierarchy, but this work will not be included in this PR.